django/core/paginator.py

```diff
@@ -24,6 +25,12 @@ class EmptyPage(InvalidPage):
 
 class Paginator(object):
 
+    error_messages = {
+        'page_not_an_integer': _('That page number is not an integer'),
+        'page_not_a_positive_integer': _('That page number is less than 1'),
+        'no_results': _('That page contains no results')
+    }
```

Comment:
In one project I need equal messages for `'page_not_an_integer'` and `'page_not_a_positive_integer'`. Of course it can be solved with translation string. But I decided to move them to `error_messages` dict.